book/src/workspaces/structure.md

```diff
@@ -6,13 +6,13 @@ contains a single package.
 
 Dist has two types of configuration:
 
-- **Workspace**:
-- **Package**: 
+- **Workspace**
```

Comment:
i was going to add brief descriptions of these next to the words- do you think it needs that?

Reply:
I think it's a good call.